<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminDeviceManager</class>
 <widget class="QWidget" name="AdminDeviceManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>923</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>130</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>添加设备</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="editButton">
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>130</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>编辑设备</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>130</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>删除设备</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>210</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>210</height>
        </size>
       </property>
       <property name="title">
        <string>筛选模块</string>
       </property>
       <widget class="QComboBox" name="typeComboBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>150</width>
          <height>30</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>50</height>
         </size>
        </property>
       </widget>
       <widget class="QComboBox" name="locationFilterComboBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>150</width>
          <height>30</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>50</height>
         </size>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>设备类型：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>设备位置：</string>
        </property>
       </widget>
       <widget class="QPushButton" name="queryButton">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>150</y>
          <width>90</width>
          <height>40</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>确定</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QTableWidget" name="deviceTable"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
